setup/docker/scripts/entrypoint.sh

```diff
@@ -16,4 +16,11 @@ JAVA_OPTS="$JAVA_OPTS -Dcom.sun.management.jmxremote.authenticate=false"
 JAVA_OPTS="$JAVA_OPTS -Dcom.sun.management.jmxremote.ssl=false"
 JAVA_OPTS="$JAVA_OPTS -Dlog4j2.configurationFile=/etc/varadhi/log4j2.xml"
 
+# If debug flag is set(1 or true) then enable remote debugging
+if [[ ! -z "$APP_DEBUG" ]]; then
+    if [[ "$APP_DEBUG" == "1" || "$APP_DEBUG" == "true" ]]; then
```

Comment:
any issue being it just set to any value ?

Reply:
no, idea was if someone sets to false or 0 or any other invalid value, we don't enable it.

Reply:
Yup. only inner check is enough